### DISCUSSION TITLE
Ignore package option

### DISCUSSION THREAD

**Discussion by nickshanks347** [microsoft/winget-cli#1358] [Ideas] [answered]
I (and other people) sometimes like their software to stay on a particular version. That means doing `winget upgrade --all` will update all packages and then the user would have to reinstall the older version. The cycle continues.

Is it possible to get an option to ignore packages on an upgrade, such as: `winget upgrade --all --ignore Microsoft.VisualStudioCode,Microsoft.AzureStorageExplorer`? Or we could have a separate command such as `winget ignore Microsoft.VisualStudioCode,Microsoft.AzureStorageExplorer` which adds the packages specified in the command to a file somewhere so the program knows which packages the user would like it to ignore.

What's everyone's thoughts on this?

**Comment by ImJoakim** [comment, marked as answer]
Hey @nickshanks347, I believe #476 is what you're asking for. If so, feel free to give that post a 👍
